fiboa_cli/datasets/nz.py

```diff
@@ -0,0 +1,56 @@
+from ..convert_utils import BaseConverter
+import pandas as pd
+
+
+class NZCropConverter(BaseConverter):
+    DATA_ACCESS = """Download manually from https://data.mfe.govt.nz/layer/105407-irrigated-land-area-raw-2020-update/"""
+
+    id = "nz"
+    short_name = "New Zealand"
+    title = "Irrigated land area"
+    description = """This dataset covers Irrigated Land. Adapted by Ministry for the Environment and Statistics
+     New Zealand to provide for environmental reporting transparency
+
+     The spatial data covers all mainland regions of New Zealand, with the exception of Nelson, which is not believed to
+     contain significant irrigated areas. The spatial dataset is an update of the national dataset that was first
+     created in 2017. The current update has incorporated data from the 2019 – 2020 irrigation season."""
```

Comment:
I'm not 100% sure how the """ works in Python, but I guess this adds additional spaces in front which would lead to a "quote" in markdown for the second paragraph.

```suggestion
    description = """
This dataset covers Irrigated Land. Adapted by Ministry for the Environment and Statistics
New Zealand to provide for environmental reporting transparency

The spatial data covers all mainland regions of New Zealand, with the exception of Nelson, which is not believed to
contain significant irrigated areas. The spatial dataset is an update of the national dataset that was first
created in 2017. The current update has incorporated data from the 2019 – 2020 irrigation season.
"""
```

Maybe we should format it as such for now?

Reply:
The spaces get added to the string, but I like the consistent indentation (otherwise it jumps back and code becomes harder to read). And then if you process the string (e.g. generate some .md file), just `strip()` the whitespaces of each line.

Reply:
But if you strip the whitespace from each line, you may also strip intentional whitespaces. e.g. in intendet lists. We should not try to correct the Markdown indentation, I think. We'll very likely don't get it right. You'd need to know how many spaces (or tabs) to strip... I think in this case we need to bite the bullet of weird Python indentation.